packages/pyright-internal/src/localization/localize.ts

```diff
@@ -186,27 +155,15 @@ function loadStringsForLocale(locale: string): StringLookupMap {
     return {};
 }
 
-function loadStringsFromJsonFile(locale: string): StringLookupMap | undefined {
+async function loadStringsFromJsonFile(locale: string): Promise<StringLookupMap | undefined> {
+    let result: StringLookupMap | undefined = undefined;
```

Comment:
I think this can be simplified by removing the result variable and returning undefined as the default for the switch case statement.

Reply:
Will fix.

I had it like that initially, changed it so I could easily log the result, then never changed it back.

Reply:
Erm, evidently half changed it back, it was more nonsensical than I realised.

Reply:
7c68d2873fd8ec91d4eb81f07a13f60bbbeed000